hist.py

```diff
@@ -0,0 +1,77 @@
+# Importing Important libraries
+from tkinter import filedialog
+from tkinter import *
+import cv2
+
+
+# Helper functions

```

Comment:
Pleas use `with` when operating on files. Please add doctests, typehints and some description about this algorithm (maybe with some links to theory).